src/avfoundation.cs

```diff
@@ -8692,6 +8970,25 @@ interface AVMutableVideoComposition {
 		[Export ("sourceSampleDataTrackIDs", ArgumentSemantic.Copy)]
 		[BindAs (typeof (int[]))]
 		NSNumber[] SourceSampleDataTrackIds { get; set; }
+
+		[Async]
```

Comment:
I am not sure we want Async here, is the completion handler really something that needs to be waited for in this context? Since it is a Create method, same for the following ones